configure.ac

```diff
@@ -1863,6 +1861,7 @@ AC_SUBST(BITCOIN_WALLET_TOOL_NAME)
 AC_SUBST(BITCOIN_MP_NODE_NAME)
 AC_SUBST(BITCOIN_MP_GUI_NAME)
 
+AC_SUBST(PATH)
```

Comment:
Ok, this seems to be a good idea anyway. :+1:

Reply:
I'm not entirely sure about this. It seems to me that the PATH needs to come from the shell/OS, not from the build system. I've never seen software do this.

Reply:
I think the point is that in this case `PATH` already comes from the build system because it's set by `config.site`, see the discussion below, https://github.com/bitcoin/bitcoin/pull/24566#issuecomment-1067800277 and further down.

Reply:
OK, I understand the reasoning then. This seems like a specific trick that we use in depends, which makes this necessary. Maybe this can be conditional on depends builds somehow?

For normal builds it seems an unexpected thing to do.

Reply:
Or maybe better: make depends explicitly override RANLIB and STRIP paths?

Reply:
@laanwj 
> Or maybe better: make depends explicitly override full RANLIB and STRIP paths, so PATH doesn't matter?

Right! Done in #24866.